cmd/kubeadm/app/apis/kubeadm/install/install_test.go

```diff
@@ -14,16 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package install
 
 import (
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/pkg/api"
+	"testing"
+
+	apitesting "k8s.io/apimachinery/pkg/api/testing"
+	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 )
 
-func addConversionFuncs(scheme *runtime.Scheme) error {
```

Comment:
this is a weird diff, but metav1 now registers these automatically in your scheme when you include it.